### PR TITLE
Default TOML File

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

We've had the DeltaV TOML files for awhile now, and it's come up that we very strongly need to update the defaults. Someone who actually knows what they're doing(I'm not a server host, I'm just the guy who codes a ton of random things), please let me know if I've gotten anything wrong here.

### CODE REVIEW

File: Resources/ConfigPresets/EinsteinEngines/default.toml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ramping_average_end_time = 180.0
+ramping_average_chaos    = 4.5
+
+[game]
+hostname           = "Default EE Server Name, please change this to your intended server name"
+desc               = "An alternative upstream that favors content intended for RP servers. If you are reading this, please change it."
+soft_max_players   = 100
+lobbyenabled       = true
+lobbyduration      = 240
+round_end_pacifist = true
```
Comment (DEATHB4DEFEAT) [inline Resources/ConfigPresets/EinsteinEngines/default.toml:18]:
```suggestion
```

File: Resources/ConfigPresets/EinsteinEngines/default.toml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+[discord]
+rich_main_icon_id = "einstein-engines"
+
+[events]
+ramping_average_end_time = 180.0
+ramping_average_chaos    = 4.5
+
+[game]
+hostname           = "Default EE Server Name, please change this to your intended server name"
```
Comment (DEATHB4DEFEAT) [inline Resources/ConfigPresets/EinsteinEngines/default.toml:13]:
```suggestion
hostname           = "Einstein Engines (change this)"
```

File: Resources/ConfigPresets/EinsteinEngines/default.toml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+[admin]
+see_own_notes   = true
+deadmin_on_join = true
+
+[discord]
+rich_main_icon_id = "einstein-engines"
```
Comment (DEATHB4DEFEAT) [inline Resources/ConfigPresets/EinsteinEngines/default.toml:6]: We need to PR the EE icon to the assets repo

File: Resources/ConfigPresets/EinsteinEngines/default.toml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+[net]
+max_connections = 1024
+tickrate = 30
+
+[netres]
+limit    = 10.0
+
+[server]
+id = "einstein-engines"
```
Comment (DEATHB4DEFEAT) [inline Resources/ConfigPresets/EinsteinEngines/default.toml:41]:
```suggestion
[build]
fork_id = "EinsteinEngines"

[server]
```

File: Resources/ConfigPresets/EinsteinEngines/default.toml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+[hub]
+tags = "lang:en-US,region:am_n_e,rp:med"
+hub_urls = "https://hub.spacestation14.com/"
+
+[ic]
+flavor_text = true
+
+[infolinks]
+discord    = "https://discord.gg/zgQVdEPJ8w"
```
Comment (DEATHB4DEFEAT) [inline Resources/ConfigPresets/EinsteinEngines/default.toml:28]:
```suggestion
discord    = "https://discord.gg/X4QEXxUrsJ"
```

File: Resources/ConfigPresets/EinsteinEngines/default.toml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+github     = "https://github.com/Simple-Station/Einstein-Engines"
+website    = "https://simplestation.org/"
+bug_report = "https://github.com/Simple-Station/Einstein-Engines/issues/new/choose"
+
+[net]
+max_connections = 1024
+tickrate = 30
+
+[netres]
+limit    = 10.0
```
Comment (DEATHB4DEFEAT) [inline Resources/ConfigPresets/EinsteinEngines/default.toml:38]:
```suggestion
limit = 10.0
```

File: Resources/ConfigPresets/EinsteinEngines/default.toml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+rich_main_icon_id = "einstein-engines"
+
+[events]
+ramping_average_end_time = 180.0
+ramping_average_chaos    = 4.5
+
+[game]
+hostname           = "Default EE Server Name, please change this to your intended server name"
+desc               = "An alternative upstream that favors content intended for RP servers. If you are reading this, please change it."
+soft_max_players   = 100
```
Comment (DEATHB4DEFEAT) [inline Resources/ConfigPresets/EinsteinEngines/default.toml:15]:
```suggestion
soft_max_players   = 60
```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Review by stellar-novas — APPROVED** [review]

**Pull request merged**